libs/vm-modules/src/ml/model/model.cpp

```diff
@@ -772,6 +774,41 @@ void VMModel::LayerAddReshape(const fetch::vm::Ptr<String> &
   }
 }
 
+void VMModel::LayerAddLoss(const fetch::vm::Ptr<String> &layer,
```

Comment:
you don't seem to bind this anywhere?
Shouldn't this be bound under the experimental bindings?

Also you should add a test using etch code (see vm_modules/test/unit/ml/jml_model_tests.cpp)

Reply:
I use it in vm_modules/benchmark/model/model.ccp in BM_AddLoss function